perf/server.py

```diff
@@ -52,14 +124,17 @@ def assign(self, test_name, cmd, threads, xs_path, stdout_file, stderr_file):
       pending_cores = running[2]
       if (start, end) == pending_cores:
         return False
-    run_cmd = self.numactl(cmd, mem, start, end)
+    if dry_run:
```

Comment:
Maybe, this can be deleted when the code passes the test?

Reply:
> Maybe, the "dry_run" setting can be deleted when the code has passed the test?

Dry run is useful for debugging without disturbing servers. Many softwares provide dry-run option, such as SEPCCPU 06/17.

Reply:
Lesson learned.